sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/recoveryservicesbackup/Generated/Models/AzureIaaSVMProtectedItemExtendedInfo.cs

```diff
@@ -32,14 +32,23 @@ public AzureIaaSVMProtectedItemExtendedInfo()
         /// AzureIaaSVMProtectedItemExtendedInfo class.
         /// </summary>
         /// <param name="oldestRecoveryPoint">The oldest backup copy available
-        /// for this backup item.</param>
+        /// for this backup item across all tiers.</param>
+        /// <param name="oldestRecoveryPointInVault">The oldest backup copy
+        /// available for this backup item in vault tier</param>
+        /// <param name="oldestRecoveryPointInArchive">The oldest backup copy
+        /// available for this backup item in archive tier</param>
+        /// <param name="newestRecoveryPointInArchive">The latest backup copy
+        /// available for this backup item in archive tier</param>
         /// <param name="recoveryPointCount">Number of backup copies available
         /// for this backup item.</param>
         /// <param name="policyInconsistent">Specifies if backup policy
         /// associated with the backup item is inconsistent.</param>
-        public AzureIaaSVMProtectedItemExtendedInfo(System.DateTime? oldestRecoveryPoint = default(System.DateTime?), int? recoveryPointCount = default(int?), bool? policyInconsistent = default(bool?))
+        public AzureIaaSVMProtectedItemExtendedInfo(System.DateTime? oldestRecoveryPoint = default(System.DateTime?), System.DateTime? oldestRecoveryPointInVault = default(System.DateTime?), System.DateTime? oldestRecoveryPointInArchive = default(System.DateTime?), System.DateTime? newestRecoveryPointInArchive = default(System.DateTime?), int? recoveryPointCount = default(int?), bool? policyInconsistent = default(bool?))
```

Comment:
There are breaking changes in the interfaces. Here are 2 options:
- Add manual codes to provide backward compatibility, see compute for reference: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations
- Or you need to bump the major version

Reply:
Added customization files for breaking changes